nixos/modules/services/mail/mailman.nix

```diff
@@ -383,6 +395,11 @@ in {
       with open('/var/lib/mailman-web/settings_local.json') as f:
```

Comment:
I didn't really get to this on time; isn't settings_local.json perfectly suited for this use case?